challenges/objects-arrays.js

```diff
@@ -48,9 +87,17 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
 
+
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+
+graduates.forEach((graduates) => {
+  let university = graduates.university;
+  return universities.push(`${university}`);
```

Comment:
you don't need to `return` anything here.  The returned value is not being used anywhere.  The key part is that you're already pushing to the `universities` array.  If it was a `.map` a return is required so the `.map` knows what it's using to build the new array.